test/e2e/specs/editor/blocks/image.spec.js

```diff
@@ -340,11 +346,15 @@ test.describe( 'Image', () => {
 		const updatedImageSrc = await image.getAttribute( 'src' );
 		expect( initialImageSrc ).not.toEqual( updatedImageSrc );
 
-		const updatedImageDataURL = await imageBlockUtils.getDataURL( image );
-		expect( initialImageDataURL ).not.toEqual( updatedImageDataURL );
+		await expect
+			.poll( () => image.boundingBox() )
+			.toMatchObject( {
+				height: 5,
+				width: 5,
+			} );
 
-		expect(
-			snapshotDiff( initialImageDataURL, updatedImageDataURL )
+		await expect(
+			await imageBlockUtils.getImageBuffer( updatedImageSrc )
```

Comment:
The `await` should be unnecessary:

```suggestion
		expect(
			await imageBlockUtils.getImageBuffer( updatedImageSrc )
```

Reply:
Good catch! Thanks!